pr-dhl-woocommerce/includes/REST_API/Parcel_DE/Client.php

```diff
@@ -149,6 +149,16 @@ public function request_info_to_request_data( array $items_info ) {
 	protected function services_mappimng( Item_Info $request_info ) {
 		$services = array();
 		foreach ( $request_info->services as $key => $service ) {
+
+			if (
+				'goGreenPlus' === $key &&
+				'DE' === PR_DHL()->get_base_country() &&
+				PR_DHL()->is_shipping_domestic( $request_info->args['shipping_address']['country'] )
+			) {
+				$services[ $key ] = ( 'true' === $service || 'yes' === $service ) ? 'true' : 'false';
```

Comment:
`( 'true' === $service || 'yes' === $service )` Just wondering why there is 2 possible values for `service` ?